Dockerfile

```
@@ -14,6 +14,12 @@ ARG NODEJS_HOME=/opt/nodejs
 
 ENV PATH=${PATH}:${SONAR_SCANNER_HOME}/bin:${NODEJS_HOME}/bin
 
+# set up local envs in order to allow for special chars (non-asci) in filenames
+ENV LC_TYPE="C.UTF-8"
```

I believe the correct key is LC_CTYPE
```suggestion
ENV LC_CTYPE="C.UTF-8"
```

---

@tom-vanbraband-sonarsource  sorry for the delay. Yep - you are right `LC_ALL` is enough. I've adjusted the PR content.